wrappers/swift/Sample/main.swift

```diff
@@ -3,8 +3,10 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+import ObjCModule // Required if Enums defined in ObjC headers are being used
```

Comment:
The enums in `ObjCModule` will almost always be used by the consumers, right? These enums should be in the Swift wrapper; as consumers of the API, we should only have to `import OneDSSwift` and all the relevant dependencies should be pulled in.

Reply:
I was trying to avoid creating duplicate enums in swift and then do the conversions for them from swift to objc.

Reply:
i have created alias for the types required to be exported in the package and now they would be available without the objc import.